src/AppBundle/Controller/MainController.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace AppBundle\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use AppBundle\Entity\Todo;
+use Symfony\Component\HttpFoundation\Request;
+
+class MainController extends Controller
+{
+    /**
+     * @Route("/main", name="main"))
+     */
+    public function mainAction()
+    {   
+        if (!($this->container->get('security.authorization_checker')->isGranted('ROLE_USER'))) {
+            return $this->redirectToRoute('homepage');
+        }
+        $user = $this->container->get('security.token_storage')->getToken()->getUser();
+
+        $todos = $user->getTodos();
+
+        return $this->render('main.html.twig', array(
+            'todos' => $todos
+        ));
+    }
+
+
+    /**
+     * @Route("/main/create", name="create_todo"))
+     */
+    public function createAction(Request $request)
+    {   $user = $this->container->get('security.token_storage')->getToken()->getUser();
+
+        $data = $request->request->get('request');
+        $todo = new Todo();
+        $todo->setBody($_POST['body']);
```

Comment:
Yes, Symfony provides a bunch of helper classes and methods so that you don't have to access the PHP superglobals yourself. It's generally considered bad practice to do so.